pytorch_blade/benchmark/TorchBench/test_torch_bench.sh

```diff
@@ -11,18 +11,24 @@
 
 # !/bin/bash
 # install dependencies
-apt install -y git git-lfs libglib2.0-0 libsndfile1 libgl1 && pip install librosa torchvision torchaudio torchtext --extra-index-url https://download.pytorch.org/whl/cu113
+python3 -m virtualenv venv --system-site-packages && source venv/bin/activate
+pip3 install -U pip
+pip3 install -q librosa torchvision torchaudio torchtext pycocotools==2.0.3 --extra-index-url https://download.pytorch.org/whl/cu113
```

Comment:
Nit: I prefer to give requirements.txt that can be configured; otherwise, the benchmark is limited to cu113. Also, you should change the index-url to https://download.pytorch.org/whl/torch_stable.html